server/api.go

```diff
@@ -161,3 +140,27 @@ func (p *Plugin) submitScore(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	w.Write(response.ToJson())
 }
+
+func getScore(selectedOption string) (int64, error) {
+	score, err := strconv.ParseInt(selectedOption, 10, 0)
+	if err != nil {
+		return 0, err
+	}
+
+	if score < 0 || score > 10 {
+		return 0, fmt.Errorf("score out of range")
```

Comment:
Nit: You can just use `errors.New`

Reply:
I used this because the file already imported `fmt`, but it's probably better to be consistent